### PR TITLE
AF-994: Persistence descriptor can not be opened

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/AF-994

Before:
before

After:
after

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Seems like a lot of white space/formatting changes. Would you mind giving a brief explanation of the issue and fix?

**Comment by karreiro** [comment]
Jenkins please retest this.

File: kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/PersistenceDescriptorEditorPresenter.java [hunk 1]
```diff
@@ -171,67 +175,81 @@ public void OnClose() {
...
     }
 
+    @Override
+    protected Supplier<PersistenceDescriptorEditorContent> getContentSupplier() {
+        return this::getUpdatedContent;
+    }
+
+    @Override
+    protected Caller<? extends SupportsSaveAndRename<PersistenceDescriptorEditorContent, Metadata>> getSaveAndRenameServiceCaller() {
+        return editorService;
```
Comment (karreiro) [inline kie-wb-common-screens/kie-wb-common-data-modeller/kie-wb-common-data-modeller-client/src/main/java/org/kie/workbench/common/screens/datamodeller/client/PersistenceDescriptorEditorPresenter.java]: The real change is here.

The _Save&Rename_ feature is being initialized by default in the KieEditor class [here](https://github.com/kiegroup/kie-wb-common/blob/master/kie-wb-common-widgets/kie-wb-metadata-widget/src/main/java/org/kie/workbench/common/widgets/metadata/client/KieEditor.java#L270). And, if we want to have that feature, we need to provide a service that implements the `SupportsSaveAndRename` to the asset that we're editing :-)

**Comment by karreiro** [comment]
@manstis I just added an inline explanation ;-)

Regarding the format changes, I'm always using that [?w=1](https://github.com/kiegroup/kie-wb-common/pull/1426/files?w=1) parameter, this helps me a lot to review PRs with a lot of white space/formatting changes. However, some PRs (like this one) this is not super effective... because some methods or fiends were re-ordered, or even some fields were changed.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Pull request merged**